backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -169,14 +169,21 @@ public void onComplete() {
             public void close() {
             }
         });
+
+        String memorySettingCommand = Optional.ofNullable(System.getenv("CI"))
```

Comment:
Shouldn't we add this somewhere for the CI then? Like maybe the Jenkinsfile?

Reply:
Your mean is `CI` ?
It is already in Jenkinsfile https://github.com/apache/james-project/pull/1689#issuecomment-1687793011

Reply:
My bad :)